tests/js/server/resilience/moving-shards-cluster.js

```diff
@@ -119,11 +119,21 @@ function MovingShardsSuite () {
     } catch (err) {
       console.error(
         "Exception for POST /_admin/cluster/cleanOutServer:", err.stack);
-      return {};
+      return {cleanedServers:[]};
+    }
+    if (res.statusCode !== 200) {
+      return {cleanedServers:[]};
     }
     var body = res.body;
     if (typeof body === "string") {
-      body = JSON.parse(body);
+      try {
+        body = JSON.parse(body);
+      } catch (err2) {
+      }
```

Comment:
err2 is ignored and never handled. Is that expected behaviour?

Reply:
Yes, if JSON.parse throws, then body is not changed and this is checked further down.

Reply:
Maybe this is not the most elegant way, but it would do the job.

Reply:
Ok, then LGTM